shell/scripts/extension/helm/charts/ui-plugin-server/templates/cr.yaml

```diff
@@ -10,3 +10,4 @@ spec:
     version: {{ (semver (default .Chart.AppVersion .Values.plugin.versionOverride)).Original }}
     endpoint: http://{{ include "extension-server.fullname" . }}.{{ .Release.Namespace }}.svc
     noCache: {{ .Values.plugin.noCache }}
+    metadata: {{ include "extension-server.pluginMetadata" . }}
```

Comment:
This needs to include the `indent 6` to properly space the range, without it will return:

```sh
$ helm template --dry-run test ./charts/test/0.1.0/
---
# Source: test/templates/cr.yaml
apiVersion: catalog.cattle.io/v1
kind: UIPlugin
metadata:
  name: test
  namespace: default
  labels: 
    helm.sh/chart: test-0.1.0
    app.kubernetes.io/name: test
    app.kubernetes.io/instance: test
    app.kubernetes.io/version: "0.1.0"
    app.kubernetes.io/managed-by: Helm
spec:
  plugin:
    name: test
    version: 0.1.0
    endpoint: https://raw.githubusercontent.com/test/test/extensions/test/0.1.0
    noCache: false
    metadata:       
catalog.cattle.io/rancher-version: ">= v2.8.4"
jordon/did-this: "true"
```

With indent:

```yaml
    metadata: {{ include "extension-server.pluginMetadata" . | indent 6 }}
```

This will produce:

```sh
$ helm template --dry-run test ./charts/test/0.1.0/
---
# Source: test/templates/cr.yaml
apiVersion: catalog.cattle.io/v1
kind: UIPlugin
metadata:
  name: test
  namespace: default
  labels: 
    helm.sh/chart: test-0.1.0
    app.kubernetes.io/name: test
    app.kubernetes.io/instance: test
    app.kubernetes.io/version: "0.1.0"
    app.kubernetes.io/managed-by: Helm
spec:
  plugin:
    name: test
    version: 0.1.0
    endpoint: https://raw.githubusercontent.com/test/test/extensions/test/0.1.0
    noCache: false
    metadata:       
      catalog.cattle.io/rancher-version: ">= v2.8.4"
      jordon/did-this: "true"
```

Reply:
@jordojordo will it be `nindent 6` or `indent 6` ? check the `metadata.labels` part 🤔 

`apiVersion: catalog.cattle.io/v1
kind: UIPlugin
metadata:
  name: {{ include "extension-server.fullname" . }}
  namespace: {{ .Release.Namespace }}
  labels: {{ include "extension-server.labels" . | nindent 4 }}
spec:
  plugin:
    name: {{ include "extension-server.fullname" . }}
    version: {{ (semver (default .Chart.AppVersion .Values.plugin.versionOverride)).Original }}
    endpoint: http://{{ include "extension-server.fullname" . }}.{{ .Release.Namespace }}.svc
    noCache: {{ .Values.plugin.noCache }}
    metadata: {{ include "extension-server.pluginMetadata" . }}`

Reply:
`indent 6` is what you would want, `nindent` will add an extra line due to it being a range.